config/migrations/mysql/000003_update_configurations_sha.up.sql

```diff
@@ -0,0 +1,52 @@
+SET @preparedStatement = (SELECT IF(
+    (
+        SELECT COUNT(*) FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE table_name = 'Configurations'
+        AND table_schema = DATABASE()
+        AND column_name = 'SHA'
+    ) > 0,
+    'SELECT 1',
+    'ALTER TABLE Configurations ADD COLUMN SHA varchar(64) DEFAULT "";'
```

Comment:
It seems like the hash output is always of size 32 bytes https://pkg.go.dev/crypto/sha256#Size. So we can use `char(64)` instead of `varchar(64)`. The database optimizes that more efficiently.